### DISCUSSION TITLE
osu!lazer no longer has separate video / storyboard toggle

### DISCUSSION THREAD

**Discussion by stravant** [ppy/osu#23174] [Ideas]
I like to play with storyboards on and low dim but lazer has no way to turn off only videos without also turning off storyboards.

Videos are almost always unplayable, vs most storyboards are designed to be playable. Thus these two should definitely have different toggles as they do in stable, otherwise you'll be forcing most people to hide a bunch of great storyboards just to hide videos.

**Comment by bdach** [comment]
The storyboard and video toggles were [intentionally merged previously, but leaving open the option of having a dropdown for finer-grained control](https://github.com/ppy/osu/pull/8167#issuecomment-596169223).
